test/test_sgg.py

```diff
@@ -115,6 +121,22 @@ def test_seasons():
     assert seasons['data'] is not None
     assert len(seasons['data']) > 0
 
+def test_special():
+    username = 'mfutch78@gmail.com'
+    password = cloud_utils.access_secret_version("593748364912", "FOREUP_MFUTCH", "latest")
+    token = foreup_utils.get_token(username, password)
+    special = foreup_utils.get_special(token, SPECIAL_TEST_CASE['COURSE_ID'], SPECIAL_TEST_CASE['TEESHEET_ID'], SPECIAL_TEST_CASE['SPECIAL_ID'])
+    assert special is not None
+    assert 'data' in special
+
+def test_specials():
+    username = 'mfutch78@gmail.com'
+    password = cloud_utils.access_secret_version("593748364912", "FOREUP_MFUTCH", "latest")
+    token = foreup_utils.get_token(username, password)
+    specials = foreup_utils.get_specials(token, SPECIAL_TEST_CASE['COURSE_ID'], SPECIAL_TEST_CASE['TEESHEET_ID'])
```

Comment:
Just out of curiosity, how many specials did this return?

Reply:
how can I check?

Reply:
You could put a print statement in your test and use pytest -rP

Reply:
---------------------------------------------- Captured stdout call ---------------------------------------------- 
Specials retrieved: 2

Reply:
not really sure if this is right or how to check ;o